recipe/meta.yaml

```diff
@@ -6,7 +6,7 @@ package:
   version: {{ version }}
 
```

Comment:
The build number should be `0`.

Reply:
done, thanks.  it is something from CF about bioconda's version